.github/workflows/main.yml

```diff
@@ -46,7 +46,7 @@ jobs:
       id: get_version
       run: |
         sudo apt install xmlstarlet
-        find -name Directory.Build.props | xargs xmlstarlet sel -N i=http://schemas.microsoft.com/developer/msbuild/2003 -t -v "concat('::set-output name=version::v',//i:VersionPrefix/text())" | xargs echo
+        find -maxdepth 1 -name Directory.Build.props | xargs xmlstarlet sel -N i=http://schemas.microsoft.com/developer/msbuild/2003 -t -v "concat('::set-output name=version::v',//i:VersionPrefix/text())" | xargs echo
```

Comment:
Could of did `run: echo "VERSION=${GITHUB_REF/refs\/tags\/v/}" >> $GITHUB_ENV` than do

`dotnet build someproject.csproj -p:Version=${VERSION}`